src/patterns/components/promo/react/code/has-image.hbs

```diff
@@ -0,0 +1,21 @@
+<SprkPromo
+  title="Title"
+  subtitle="Subtitle"
+  additionalClasses="sprk-o-Stack--split@s"
+  cta="button"
+  ctaText="Learn More"
+  ctaHref="https://www.sparkdesignsystem.com"
+  ctaAnalytics="promo-cta-5-analytics"
+  ctaIdString="promo-cta-5"
+  imgAlt="placeholder"
+  imgSrc="https://sparkdesignsystem.com/assets/toolkit/images/flower.jpg"
```

Comment:
Is there a reason there's an extra space here?

Reply:
Because my editor isn't highlighting them for me 😫

Is it a linter that is supposed to be catching those on my end?